src/main/java/com/google/ehub/data/EntertainmentItemDatastore.java

```diff
@@ -63,10 +68,14 @@ public void addItemToDatastore(EntertainmentItem item) {
 
     // Unique Id is created by Datastore so it's not added as a property.
     itemEntity.setProperty(DISPLAY_TITLE_PROPERTY_KEY, item.getTitle());
-    itemEntity.setProperty(NORMALIZED_TITLE_PROPERTY_KEY, item.getTitle().toLowerCase());
+    itemEntity.setProperty(NORMALIZED_TITLE_PROPERTY_KEY,
+                           item.getTitle().toLowerCase());
     itemEntity.setProperty(DESCRIPTION_PROPERTY_KEY, item.getDescription());
     itemEntity.setProperty(IMAGE_URL_PROPERTY_KEY, item.getImageUrl());
     itemEntity.setProperty(RELEASE_DATE_PROPERTY_KEY, item.getReleaseDate());
```

Comment:
Do we still need to save RELEASE_DATE_PROPERTY_KEY to the datastore? One field for the release date should be enough. You can convert RELEASE_DATE_TIMESTAMP_MILLIS_PROPERTY_KEY to any formatted date using a Utils function.

Reply:
Ok I removed the release date property and instead kept the timestamp one with a Utils method.